package.json

```diff
@@ -30,6 +30,12 @@
     "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
     "test:ember": "ember test"
   },
+  "overrides": {
+    "ember-power-select": "$ember-power-select"
+  },
+  "overridesNotes": {
+    "ember-power-select": "Appuniversum needs to widen the range so v7 is supported. Once it does we can remove the override again."
```

Comment:
I opened a PR in appuniversum, so I can remove it if that gets merged and released soonish.